src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -148,12 +148,15 @@ private static void OnUIElementKeyUp(object sender, KeyRoutedEventArgs e)
 			// handle enter command
 			CommandExtensions.TryInvokeCommand(host, CommandExtensions.GetCommandParameter(host) ?? GetInputParameter());
 
+#if HAS_UNO
```

Comment:
@ajpinedam What happens on Windows?

Reply:
i guess there are issue with the `InputPane` class on winui-specifically?
@ajpinedam maybe this can help you? https://github.com/microsoft/microsoft-ui-xaml/issues/6291#issuecomment-1311614806

Reply:
Yeah InputPane.GetForCurrentView won't work in winappsdk

Reply:
we should probably have an injection point for the winui-windows: `WindowHelper.Instance {get; set;}` (`Window.Current` is null and get-only)
@kazo0 we should also make a pass over various `.GetForCurrent(View|Thread)` methods usage and see which fails and provide an alternative codepath in toolkit

Reply:
@Xiaoy312  @kazo0 I am a bit unsure about what I should do next for this PR. Please advise

Reply:
@ajpinedam You don't have to do anything. I'll approve the PR as is.
I'll open another issue for the problem in this discussion, and addresses that separately.